.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -24,6 +24,9 @@ body:
         - [ ] build <https://build.libelektra.org/job/PermaplanT-Release/>
         - [ ] git tag -s vX.X.X
         - [ ] git push --tags
+
+        ### After Release
+
         - [ ] create release PR to pump versions and new section in Changelog
         - [ ] write announcement in [nextcloud](https://nextcloud.markus-raab.org/nextcloud/)
```

Comment:
This better afterwards next time so that people correctly know when they actually can rebase. (usually it is only a matter of minutes but who knows).